doc/news/_preparation_next_release.md

```diff
@@ -322,6 +324,10 @@ We are currently working on following topics:
 
 <<`scripts/git-release-stats 0.9.VER-1 0.9.<<VERSION>>`>>
 
+## Finished Thesis
+
+- [Klemens Böswirth](https://www.libelektra.org/ftp/elektra/publications/boeswirth2019highlevel.pdf)
```

Comment:
You are welcome to write a bit about your thesis here (as taster so that people click on the link).